src/transformStateWithClones.js

```diff
@@ -64,8 +64,34 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const arrOfStates = [];
+  let copyOfStateObj = Object.assign({}, state);
```

Comment:
```suggestion
  const statesHistory = [];
  let stateCopy = Object.assign({}, state);
```
Let's use more meaningful and readable names for your variables in the future